third-party/chromium/data/verify_signed_data/ours/ecdsa-prime256v1-sha256-spki-inside-spki.pem

```diff
@@ -0,0 +1,36 @@
+This is a copy of ecdsa-prime256v1-sha256-spki-inside-spki.pem, but
```

Comment:
Thanks for including these textual headers & the `openssl` cmds. Much appreciated.